website/docs/tutorial/dao/TributeUI.md

```diff
@@ -100,17 +106,23 @@ Then open the config file: _subgraph/config/subgraph-config.json_, remove all th
     "daoFactoryAddress": "0x...",
     "daoFactoryStartBlock": ...,
     "GITHUB_USERNAME": "<YOUR_GITHUB_USERNAME>",
-    "SUBGRAPH_NAME_OR_SLUG": "tribute-dao-tutorial"
-  }
+    "SUBGRAPH_NAME": "tribute-dao-tutorial"
+  },
 ]
 ```
 
-In the rinkeby deployment logs at _tribute-contracts/logs/rinkeby-deploy.log_ search by **DaoFactory** and copy the **contract address** and **block number**, set the values to **daoFactoryAddress** and **daoFactoryStartBlock** respectivelly. Finally, set your Github username to **GITHUB_USERNAME**, it must be the same Github account that you used to connect to thegraph.com.
+In the rinkeby deployment logs at _tribute-contracts/logs/rinkeby-deploy.log_ search by **DaoFactory** and copy the **contract address** and **block number**, set the values to **daoFactoryAddress** and **daoFactoryStartBlock** respectivelly. 
```

Comment:
Little typo (I think that was me) `respectively` not `respectivelly`

Reply:
cool, thanks! just updated.